rules/dropbox_rules/dropbox_user_disabled_2fa.yml

```diff
@@ -0,0 +1,102 @@
+AnalysisType: rule
```

Comment:
if this is removing 2FA for themselves, then agree that's low. If this is removing 2FA for all users, then I think the severity should be higher.